sites/partners/next.config.js

```diff
@@ -41,17 +41,18 @@ module.exports = withBundleAnalyzer(
       backendProxyBase: BACKEND_PROXY_BASE,
       listingServiceUrl: BACKEND_API_BASE + LISTINGS_QUERY,
       idleTimeout: process.env.IDLE_TIMEOUT,
-      showSmsMfa: (process.env.SHOW_SMS_MFA || "TRUE") === "TRUE", // SMS on by default
+      showSmsMfa: !process.env.SHOW_SMS_MFA || process.env.SHOW_SMS_MFA === "TRUE" ? "TRUE" : "", // SMS on by default
```

Comment:
The updates to this file are because Next no longer allows booleans in the config. All values that are booleans now have either "TRUE" or an empty string.